src/cmd-fetch

```diff
@@ -46,4 +50,21 @@ if [ $# -ne 0 ]; then
 fi
 
 prepare_build
-runcompose --download-only
+
+lock_arg=
+if [ -n "${UPDATE_LOCKFILE}" ]; then
+    # Put this under tmprepo so it gets automatically chown'ed if needed
+    lock_arg="--ex-write-lockfile-to=${tmprepo}/tmp/manifest-lock.json"
+elif [ -f "${manifest_lock}" ]; then
+    lock_arg="--ex-lockfile=${manifest_lock}"
+    echo "Using lockfile ${manifest_lock}"
```

Comment:
can we add an `echo "Downloaded rpms will be controlled by the lockfile"`

Reply:
I changed it to `Fetching RPMs from lockfile ...` to be a bit less wordy :)